<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1216</width>
    <height>529</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(174, 193, 255)</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lb_Header">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>541</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>Добро пожаловать в базу данных Аэрофлота.</string>
    </property>
   </widget>
   <widget class="QTableView" name="tw_flight">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>1191</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 85, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_searchElem">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>380</y>
      <width>140</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 255);
color:rgb(255, 255, 255);</string>
    </property>
    <item>
     <property name="text">
      <string>Номер рейса</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Бортовой номер</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Время вылета</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Время прилёта</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Дата полёта</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Город вылета</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Город прилёта</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="le_searchForElem">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>140</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 85, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_search">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>380</y>
      <width>140</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 255);
color:rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Найти</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_sortAscending">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>380</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 255);
color:rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Отсортировать по возрастанию</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_change">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>380</y>
      <width>140</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 255);
color:rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Заменить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_delete">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>380</y>
      <width>140</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 255);
color:rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_flightNumber">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>140</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 85, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_boardNumber">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>440</y>
      <width>140</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 85, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_arrivalTime">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>440</y>
      <width>140</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 85, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_departureTime">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>440</y>
      <width>140</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 85, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_dateOfFlight">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>440</y>
      <width>140</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 85, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="ll_flightNumber">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>Номер рейса</string>
    </property>
   </widget>
   <widget class="QLabel" name="ll_boardNumber">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>410</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>Бортовой номер</string>
    </property>
   </widget>
   <widget class="QLabel" name="ll_arrivalTime">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>410</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>время вылета</string>
    </property>
   </widget>
   <widget class="QLabel" name="ll_departureTime">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>410</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>Время прилёта</string>
    </property>
   </widget>
   <widget class="QLabel" name="ll_dateOfFlight">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>410</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>Дата полёта</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_add">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>440</y>
      <width>140</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 255);
color:rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_cityOfDeparture">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>440</y>
      <width>140</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 85, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_cityOfArrival">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>440</y>
      <width>140</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 85, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="ll_cityOfDeparture">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>410</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>Город вылета</string>
    </property>
   </widget>
   <widget class="QLabel" name="ll_cityOfArrival">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>410</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>Город прилёта</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_export">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>380</y>
      <width>140</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 255);
color:rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Экспортировать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_import">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>410</y>
      <width>140</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 255);
color:rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Импортировать</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1216</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
